### PR TITLE
Reconfigures 'plastic_molding' recipes to be more accurate

### LINKED ISSUE #69598: Retractable telescope has 2 copies of required tools
### Describe the bug

Retractable telescope recipe has 2 copies of the same tool requirement. I also noticed the "coilgun" has 2 copies of required tools too. and "tonfa"

### Attach save file

N/A trivially reproducible

### Steps to reproduce

1. look in crafting window for retractable telescope or coilgun or tonfa
2. 2 copies of tool requirement

### Expected behavior

One copy of tool requirement

### Screenshots

Untitled

### Versions and configuration

- OS: Windows
    - OS Version: MINGW/CYGWIN/MSYS2 on unknown Windows version
- Game Version: eca466b [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Slowdown Fungal Growth [no_fungal_growth],
    Stats Through Skills [StatsThroughSkills]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by TealcOneill**
#### Summary
Bugfixes "Removes unnecessary use of 'surface_heat' from recipes with 'plastic_molding'" 

#### Purpose of change
Closes #69598

#### Describe the solution
In all of these recipes the only real possible use for 'surface_heat' was for molding the plastic but 'plastic_molding' already has 'surface_heat' built into it. As such I removed the unnecessary extra use of it. 
I also balanced the charges of 'plastic_molding' to get it closer to the 1 use per 70g of plastic as possible. 
I fixed small recipes with the assumption of some waste of plastic and so didn't account for the weight of the polycarbonate sheet from the 'plastic_molding' requirement.
For bigger items like the 30gallon drum and plastic boat hulls I did account for the weight of the sheets since it seemed like they would make up a larger amount of the item being easier to turn into a large flat/curved surface.

#### Describe alternatives you've considered
Using C++ to add the charges of the two requirements together as Guardian suggested

#### Testing
Just json changes. Loaded it up and looked to make sure it was correct.

#### Additional context
image

### CODE REVIEW

**Comment by TealcOneill** [comment]
I thought I did a really careful job on this and it didn't really need linting but obviously that's not the case. I'll go through and lint all the files tonight after I get back from work.

**Comment by TealcOneill** [comment]
At this point I'm not sure what the issue is. It's definitely a related failure because it breaks in the middle of crafting something I changed but the linter shows no errors and I can't find anything else that would cause it.

**Comment by TealcOneill** [comment]
I'm going to talk to someone on the discord tomorrow to get a hand with this. I'm not absolutely sure it's caused by my changes to the carver since it did finish making it. The first Test test commit is good, the recipe definitely needs that but the second one I think needs to be reverted.

File: data/json/items/tool/misc.json [hunk 1]
```diff
@@ -688,7 +688,7 @@
     "type": "TOOL",
     "name": { "str": "retractable telescope" },
     "description": "A homemade collapsible telescope, perfect for a pirate.  Too large and inaccurate to use as a weapon scope, but carrying it in your inventory will double the distance that is mapped around you during your travels.",
-    "weight": "500 g",
+    "weight": "330 g",
```
Comment (Maleclypse) [inline data/json/items/tool/misc.json:691]:
```suggestion
    "weight": "330 g"
```

File: data/json/items/tool/misc.json [hunk 2]
```diff
@@ -688,7 +688,7 @@
     "type": "TOOL",
     "name": { "str": "retractable telescope" },
     "description": "A homemade collapsible telescope, perfect for a pirate.  Too large and inaccurate to use as a weapon scope, but carrying it in your inventory will double the distance that is mapped around you during your travels.",
-    "weight": "500 g",
+    "weight": "330 g"
```
Comment (Maleclypse) [inline data/json/items/tool/misc.json:691]:
```suggestion
    "weight": "330 g",
```

**Comment by Maleclypse** [comment]
I've made changes to kick the tests.  If they still come back with charges going wrong it's probably something in this PR.

**Comment by Maleclypse** [comment]
```
(~[slow] ~[.],starting_items)=> 12:00:00PM: You wield your in progress electric carver (off) (0%).
(~[slow] ~[.],starting_items)=> 12:00:00PM: Wielding - in progress electric carver (off) (0%)
(~[slow] ~[.],starting_items)=> 12:00:00PM: You start working on the in progress electric carver (off) (0%).
(~[slow] ~[.],starting_items)=> 12:00:00PM: You wield your electric carver (off).
(~[slow] ~[.],starting_items)=> 12:00:00PM: Wielding - electric carver (off)
(~[slow] ~[.],starting_items)=> 12:00:00PM: You craft electric carver (off) from memory.
```
Electric carver uses plastic molding the test confirms that electric carver requires the number of charges set in the test.  You'll need to adjust the number the test is looking for.

**Comment by TealcOneill** [comment]
I misunderstood how the multiple tools with enough charges section worked when I changed it in the 'Testing Test 2' commit. 
I think that's where Basic Build is failing now so I'll fix that and let the test run to see where the actual issue is.
Thanks for taking a look Maleclypse 👌

**Comment by TealcOneill** [comment]
I think that fixed everything! All failing tests look unrelated

**Pull request merged**